time time
uint8 imuId
AccelGyroMag data
